docs/release-notes/8.17.asciidoc

```diff
@@ -0,0 +1,123 @@
+
+====
+// end::known-issue[]
+
+// tag::known-issue[]
+[discrete]
+.Manually running custom query rules with suppression could suppress more alerts than expected
+[%collapsible]
+====
+*Details* +
+On November 12, 2024, it was discovered that manually running a custom query rule with suppression could incorrectly inflate the number of suppressed alerts. 
+
+====
+// end::known-issue[]
+
+[discrete]
+[[features-8.17.0]]
+==== New features
+* Adds a signature option for trusted applications on macOS ({kibana-pull}197821[#197821]).
+
+[discrete]
+[[enhancements-8.17.0]]
+==== Enhancements
+* Checks user permissions before initializing the entity engine ({kibana-pull}198661[#198661]).
+
+[discrete]
+[[bug-fixes-8.17.0]]
+==== Bug fixes
+* Clears the error on the second entity engine initialization ({kibana-pull}202903[#202903]).
+* Modifies the empty state message that appears when installing prebuilt rules ({kibana-pull}202226[#202226]).
+* Rejects CEF logs from Automatic Import and redirects you to the CEF integration instead ({kibana-pull}201792[#201792]).
+* Disables the **Install All** button on the **Add Elastic Rules** page while rules are being installed ({kibana-pull}201731[#201731]).
+* Turns off the **Add note** button in the alert details flyout if you're lacking adequate privileges ({kibana-pull}201707[#201707]).
+* Removes fields with an `@` from the script processor ({kibana-pull}201548[#201548]).
+* Fixes an issue that could interfere with Knowledge Base ({kibana-pull}201175[#201175]).
+* Fixes a bug with threshold rules that prevented cardinality details from appearing ({kibana-pull}201162[#201162]).
+* Fixes a bug that caused the risk engine to get stuck in the `Installing` status if the default Security data view didn't exist. With this fix, the engine now correctly reports the `Error` state ({kibana-pull}201140[#201140]).
+* Fixes an issue that prevented you from successfully importing TSV files with Asset Criticality data if you're on Windows ({kibana-pull}199791[#199791])
+* Fixes Asset Criticality index issue when setting up entity engines concurrently ({kibana-pull}199486[#199486]).
+* Fixes a bug that prevented the save notification from displaying on duplicated Timelines with changes ({kibana-pull}198652[#198652]).
+* Improves the flow for the Insights section in the alert details flyout ({kibana-pull}197349[#197349]).
+* Fixes a bug where the `@timestamp` field wouldn't update upon Asset Criticality soft delete ({kibana-pull}196722[#196722]).
+* Improves {elastic-defend} for Linux endpoints by enabling process information enrichment for file and network events when process events are disabled.
+* Improves {elastic-defend} by refactoring the kernel driver to work around the CrowdStrike `CRITICAL_PROCESS_DIED` bug check (BSOD).
+* Fixes an {elastic-defend} bug where the Linux system call (setsid) wasn't properly gathered for RHEL 9/CentOS Stream 9 process events.
```

Comment:
```suggestion
* Fixes an {elastic-defend} bug where the Linux system call (setsid) wasn't properly gathered for RHEL 9/CentOS Stream 9 process events.
* Fix an issue where {elastic-defend} can enter an infinite loop if an external application opens and retains handles to files within {elastic-defend}'s directory while it is processing a `get-file` response action. This can result in {elastic-defend} flooding Elasticsearch with documents until the handles are closed.
```